src/analyzer/protocol/bittorrent/BitTorrentTracker.cc

```diff
@@ -344,12 +368,17 @@ void BitTorrentTracker_Analyzer::EmitRequest(void)
 bool BitTorrentTracker_Analyzer::ParseResponse(char* line)
 	{
 	static bool initialized = false;
-	static regex_t r_stat, r_hdr;
+	static RE_Matcher re_stat("^HTTP/[0123456789.]* ");
```

Comment:
Same here, plus you can use the `0-9` range as you properly suggested for `re_get_end` above
```suggestion
	static RE_Matcher re_stat("^HTTP/[0-9\.]* ");
```